src/tensorflow/tensorflow-rnnlm-lib.h

```diff
@@ -0,0 +1,156 @@
+// tensorflow-rnnlm-lib.h
```

Comment:
I don't like '-lib' appearing in the name of these C++ files, it could cause some confusion.  Just remove the -lib.